tools/termux-installation.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: page
+nav_order: [[no idea what to set this to]]
```

Comment:
```suggestion
nav_order: 2
```

That would make it appear right below owotracker.
You'll need to modify lime-slimes.html to 3 and shadowpc.html to 4 tho